platform/windows/godot_win.cpp

```diff
@@ -137,8 +137,13 @@ int widechar_main(int argc, wchar_t** argv) {
 
 	Error err = Main::setup(argv_utf8[0], argc - 1, &argv_utf8[1]);
 
-	if (err!=OK)
+	if (err!=OK) {
+		for (int i=0; i<argc; ++i) {
+			delete[] argv_utf8[i];
```

Comment:
`delete[]` is present in C++98 standard. I don't see the point of freeing the memory just before exiting though.